documentation/PouchManager.md

```diff
@@ -0,0 +1,55 @@
+## Pouch Manager:
+
+When GPII runs in the development configuration, the pouch manager manages the pouchDB that is used as the authorization server data storage. It provides:
+* A pouchDB express server that allows GPII to integrate with CouchDB-like PouchDB. This express server makes use of gpii-pouchdb repo: https://github.com/gpii/gpii-pouchdb/ to provide the CouchDB-like API. By default, it listens on port 8058;

```

Comment:
A pouchdb-express server that allows the GPII to integrate with PouchDB exposed over a CouchDB-like HTTP API.